modes.go

```diff
@@ -413,11 +408,29 @@ func DoReadsFromTable(table string, session *gocql.Session, threadResult *result
 		request = fmt.Sprintf("SELECT %s FROM %s.%s WHERE pk = ? AND ck >= ? %s LIMIT %d", selectFields, keyspaceName, table, orderBy, rowsPerRequest)
 	}
 
```

Comment:
Those lines were using `keyspaceName` before but this variable wasn't passed as a parameter. Where was it coming from before and why to you need to pass it as a parameter now?

Reply:
removed